fragmentino/io.py

```diff
@@ -67,15 +67,16 @@ def write(self, symbols, xyz, comment=""):
             Optional comment for xyz-file comment line
 
         """
-
-        np.savetxt(
-            self.file_name,
-            np.c_[symbols, xyz],
-            encoding="utf-8",
-            fmt="%s %.20s %.20s %.20s",
-            header="\n".join([str(xyz.shape[0]), comment]),
-            comments="",
-        )
+        with open(self.file_name, "w") as f:
+            f.write(str(xyz.shape[0]) + "\n")
+            f.write(comment + "\n")
+            for symbol, pos in zip(symbols, xyz):
+                f.write(
+                    "{} {:15.10f} {:15.10f} {:15.10f}".format(
```

Comment:
Why not use f-strings ([RealPython guide](https://realpython.com/python-f-strings/))? I believe it can be written like this

```python
f"{symbol} {pos[0]:15.10f} {pos[1]:15.10f} {pos[2]:15.10f}\n"
```